spec/models/atmosphere/appliance_spec.rb

```diff
@@ -151,6 +145,55 @@
     end
   end
 
+  context 'when instantiated' do
+    let(:appliance_type) { create(:appliance_type) }
+    let(:appliance_set) { create(:appliance_set) }
+    let(:appliance) { create(:appliance,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.